src/api-next/api-next.ts

```diff
@@ -1175,7 +1181,10 @@ export class IBApiNext {
           [[EventName.headTimestamp, this.onHeadTimestamp]],
           `${JSON.stringify(contract)}:${whatToShow}:${useRTH}:${formatDate}`
         )
-        .pipe(map((v: { all: string }) => v.all))
+        .pipe(map((v: { all: string }) => v.all)),
+      {
+        defaultValue: "",
```

Comment:
can we just send undefined?

Reply:
We would need to change function signature to 
```
 getHeadTimestamp(
    contract: Contract,
    whatToShow: string,
    useRTH: boolean,
    formatDate: number
  ): Promise<string | undefined> {
```
which is kind of pointless, because it will never happen (you need to check for !== undefined, even thou it will never be undefined).
There is only a onHeadTimestamp callback and when it is invoked, we emit a value and complete. So the default is kind of 'dummy' as it will never be case, it's only relevant for functions that have like onOpenOrder and onOpenOrderEnd callback. Here the onOpenOrder emits value and onOpenOrderEnd complete, so it can actually happen that we fallback to default, while here it can't as emit+complete is one callback.